docs/reference/query-dsl/script-score-query.asciidoc

```diff
@@ -92,17 +92,18 @@ cosine similarity between a given query vector and document vectors.
         "match_all": {}
       },
       "script": {
-        "source": "cosineSimilarity(params.queryVector, doc['my_dense_vector'])",
+        "source": "cosineSimilarity(params.queryVector, doc['my_dense_vector']) + 1.0" <1>,
         "params": {
-          "queryVector": [4, 3.4, -0.2]  <1>
+          "queryVector": [4, 3.4, -0.2]  <2>
         }
       }
     }
   }
 }
 --------------------------------------------------
 // NOTCONSOLE
-<1> To take advantage of the script optimizations, provide a query vector as a script parameter.
+<1> The script adds 1.0 to the cosine similarity to prevent the score from being negative.
```

Comment:
There seems to be no previous mentioning of the restriction on scores having to be positive on this docs page. I think it would be useful to add one in a more prominent position (maybe even in the opening section) and explain this limitation (and maybe why it is there).